drivers/usbdev/cdcacm.c

```diff
@@ -2354,6 +2358,10 @@ static int cdcuart_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 
         termiosp->c_cflag |= (priv->iflow) ? CRTS_IFLOW : 0;
```

Comment:
@pkarashchenko - I here what your saying. This pattern is the same in other drivers.  It could be a sperate PR to fix them all. This code has been tested and works.

Reply:
I am fine with the change now, @pkarashchenko do you have more concern?

Reply:
Change is fine to me. Let's merge and do any other change as a follow up PR.